docs/docs/gallery.md

```diff
@@ -0,0 +1,7 @@
+# Description 
+
```

Comment:
Can you add a sub-section heading here? In the future, I think we want to add more movies to this page, so it would be useful to indicate that this movie is one of many to come.

Reply:
Yes! So, may be I should make a gallery folder with first file being just for qedIiii?

Reply:
Now it goes - Simulation Gallery-->Hydro-->QED-->Solar etc.